human.txt

```diff
@@ -31,8 +32,8 @@ jsstrn
 justyn-clark
 kenhrmt
 kylecho
-Leo-Mu
 leo424y
+Leo-Mu
```

Comment:
Why have you moved this? 😱

Reply:
Oops! I was wrong. Thanks for the sort clarification